src/main/java/com/microsoft/azure/msalwebsample/AuthPageController.java

```diff
@@ -130,18 +129,18 @@ private String getUserNamesFromGraph(String accessToken) throws Exception {
         }
 
         JSONObject responseObject = HttpClientHelper.processResponse(responseCode, response);
-        JSONArray users = JSONHelper.fetchDirectoryObjectJSONArray(responseObject);
-
-        // Parse JSON to User objects, and append user names to string
-        StringBuilder builder = new StringBuilder();
-        for (int i = 0; i < users.length(); i++) {
-            JSONObject thisUserJSONObject = users.optJSONObject(i);
-            User user = new User();
-            JSONHelper.convertJSONObjectToDirectoryObject(thisUserJSONObject, user);
-            builder.append(user.getUserPrincipalName());
-            builder.append("<br/>");
-        }
-        return builder.toString();
+//        JSONArray users = JSONHelper.fetchDirectoryObjectJSONArray(responseObject);
```

Comment:
If the code is not being used, we should delete it as opposed to commenting it out

Reply:
yes commited it by mistake, deleted.